### ISSUE TITLE
Classification

### ISSUE THREAD

**Issue by Lachlan1478** [Lachlan1478/LSTM_Regression#16]

**Comment by Lachlan1478** [comment]
Potential Code:
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import LSTM, Dense

# Assume your original LSTM model looks something like this:
model = Sequential()
model.add(LSTM(64, input_shape=(10, 1)))
model.add(Dense(1))

# Now suppose you have a new target variable y with three classes: -1, 0, and 1.
# You can convert y to a one-hot encoded format:
from tensorflow.keras.utils import to_categorical
y = to_categorical(y + 1, num_classes=3)  # y is assumed to be a 1D array of -1, 0, or 1

# Add a classification layer to the output of the LSTM layer:
model = Sequential()
model.add(LSTM(64, input_shape=(10, 1)))
model.add(Dense(64, activation='relu'))
model.add(Dense(3, activation='softmax'))  # 3 is the number of classes

# Train the model using categorical cross-entropy loss and the Adam optimizer:
model.compile(loss='categorical_crossentropy', optimizer='adam')
model.fit(X, y, epochs=10, batch_size=32)
